src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ContentPresenter/ContentPresenter_NativeEmbedding.xaml

```diff
@@ -18,7 +18,7 @@
 	<skia:ScrollViewer Height="300">
```

Comment:
Can you add more steps to the validation description, so that we know what to look for after the changes of this PR?